.travis.yml

```diff
@@ -1,11 +1,19 @@
+arch:
+    - amd64
+    - arch64
```

Comment:
The rest of the file adds a blank line between these definitions, so I think it would be consistent to add a blank line between arch and language as well.